pkg/joinserver/config.go

```diff
@@ -22,5 +22,6 @@ type Config struct {
 	Keys                          KeyRegistry                          `name:"-"`
 	ApplicationActivationSettings ApplicationActivationSettingRegistry `name:"-"`
 	JoinEUIPrefixes               []types.EUI64Prefix                  `name:"join-eui-prefix" description:"JoinEUI prefixes handled by this JS"`
+	DefaultJoinEUI                types.EUI64                          `name:"default-join-eui" description:"the default JoinEUI for this JS"`
```

Comment:
```suggestion
	DefaultJoinEUI                types.EUI64                          `name:"default-join-eui" description:"Default JoinEUI for this Join Server"`
```

Reply:
Please also change to "Join Server" above. We avoid shortening NS/AS/JS/... in places that users see.